### ISSUE TITLE
Fatal Error during the Setup Upgrade

### ISSUE THREAD

**Issue by ajith107** [ctasca/mage-bundle#2]
#### Description

- After installing the extension, while trying to run the `bin/magento setup:upgrade` command, It return the following fatal error

```
PHP Fatal error:  Declaration of Ctasca\MageBundle\Logger\Logger::info($message, array $context = []): bool must be compatible with Monolog\Logger::info($message, array $context = []): void in /var/www/html/mage-ee-244/vendor/ctasca/mage-bundle/Logger/Logger.php on line 36
```

#### Environment Details

-  Magento 2.4.4-p1
- PHP 8.1.9

**Comment by ctasca** [comment]
@ajith107 Hi...Thanks for your report.

I have had a look into this and seems /Monolog/Logger changed the return type from bool to void...

In my dev environment the log method for /Monolog/Logger is:

```php
    /**
     * Adds a log record at an arbitrary level.
     *
     * This method allows for compatibility with common interfaces.
     *
     * @param  mixed   $level   The log level
     * @param  string $message The log message
     * @param  array  $context The log context
     * @return bool   Whether the record has been processed
     */
    public function log($level, $message, array $context = array())
    {
        $level = static::toMonologLevel($level);

        return $this->addRecord($level, $message, $context);
    }
```

while on their latest release the log method is:

```php
/**
     * Adds a log record at an arbitrary level.
     *
     * This method allows for compatibility with common interfaces.
     *
     * @param mixed             $level   The log level (a Monolog, PSR-3 or RFC 5424 level)
     * @param string|Stringable $message The log message
     * @param mixed[]           $context The log context
     *
     * @phpstan-param Level|LogLevel::* $level
     */
    public function log($level, string|\Stringable $message, array $context = []): void
    {
        if (!$level instanceof Level) {
            if (!is_string($level) && !is_int($level)) {
                throw new \InvalidArgumentException('$level is expected to be a string, int or '.Level::class.' instance');
            }

            if (isset(self::RFC_5424_LEVELS[$level])) {
                $level = self::RFC_5424_LEVELS[$level];
            }

            $level = static::toMonologLevel($level);
        }

        $this->addRecord($level, (string) $message, $context);
    }
```
I'll have a look into this and make the fix. 

Thanks again

**Comment by ctasca** [comment]
Closed by mistake

**Comment by ctasca** [comment]
@ajith107 I have made a fix in version v3.5.2-BETA... Can you confirm the issue has been resolved? 
Thank you

**Comment by ajith107** [comment]
@ctasca I can confirm that the fix is working as expected. Thanks a lot
